.travis.yml

```diff
@@ -88,5 +88,5 @@ script:
     # test if all the sources are conform to the forUncrustifySources.cfg config file
   - if [ "x${BUILD_TYPE}" == "xrelease" ]; then cd ../; ./scripts/Run_uncrustify_for_sources.sh; fi
     # test if uncrustify runs for (some) command line options
-  - if [ "x${BUILD_TYPE}" == "xrelease" ]; then ./tests/cli/test_cli_options.sh; fi
+  - if [ "x${BUILD_TYPE}" == "xrelease" ]; then cd tests/cli; ./test_options_cli.py; fi
```

Comment:
Have you tested that? It looks like this should fail as you are in the `uncrustify/builld` dir (lines 57 & 89).
If you actually need that cd, I'd expect that you jump back (`cd -`) in the end. The same applies for line 89.